Source/WebCore/html/HTMLLinkElement.cpp

```diff
@@ -167,9 +166,13 @@ void HTMLLinkElement::setDisabledState(bool disabled)
 void HTMLLinkElement::parseAttribute(const QualifiedName& name, const AtomString& value)
 {
     if (name == relAttr) {
-        m_relAttribute = LinkRelAttribute(document(), value);
+        auto newRelAttribute = LinkRelAttribute(document(), value);
```

Comment:
I would be tempted to use the word “parsed” in this local variable name. In fact, the class `LinkRelAttribute` annd the data member `m_relAttribute` also should probably should have the word “parsed” in their names, since they do not contain the actual value of the rel attribute! No need to change this, just noticing in passing.

Reply:
I'd change this local variable to `parsedRel`.